tests/diag.sh

```diff
@@ -1933,6 +1933,99 @@ init_elasticsearch() {
 	curl --silent -XDELETE localhost:${ES_PORT:-9200}/rsyslog_testbench
 }
 
+omhttp_start_server() {
+    # Args: 1=port 2=server args
+    # Args 2 and up are passed along as is to omhttp_server.py
+    omhttp_server_py=$srcdir/omhttp_server.py
+    if [ ! -f $omhttp_server_py ]; then
+        echo "Cannot find ${omhttp_server_py} for omhttp test"
+        error_exit 1
+    fi
+
+    if [ "x$1" == "x" ]; then
+        omhttp_server_port="8080"
+    else
+        omhttp_server_port="$1"
+    fi
+
+    dep_work_dir=$(readlink -f .dep_wrk)
+    # Create work directory for parallel tests
+    omhttp_work_dir=${dep_work_dir}/omhttp_$RSYSLOG_DYNNAME
```

Comment:
just general thought: `.dep*` is for external dependencies. These are NOT automatically cleaned up. It may be better to use `mkdir $RSYSLOG_DYNNAME/mydir` here - but it's up to discussion. The latter is cleaned up automatically at the end of successful test.